### DISCUSSION TITLE
The end date of the Expedition

### DISCUSSION THREAD

**Discussion by leodium** [cwmonkey/nms-expeditions#1] [Ideas]
**EndTimeUTC : "EpochTime"**

Is it possible to set it to  

option 01 >
+6 weeks from the date on which the json file is uploaded to your GitHub? That would give the same timeframe for completion as "normal".

Option 02 >
+12 months from the date on which you upload the json file to your GitHub? This would leave an "infinite" amount of time?

**Comment by cwmonkey** [comment]
Ah, good idea. I can just make this a customization option. Something like `EndTimeUTC` = `now()` + 6 weeks, 6 months, 12 months, etc where `now()` will be the current date in the web browser.

**Reply by cwmonkey** [reply]
ok, done in 723d73c

**Reply by cwmonkey** [reply]
I could add like 1 day and 3 days for people who want to challenge themselves to complete it quickly. Or like hours I guess, heh. Either way the groundwork is there to expand it if desired.

**Comment by leodium** [comment]
It's your project... It's up to you! But indeed, the basics are there to challenge yourself to the maximum! Great job anyway!
